packages/nc-gui/components/nc/Tooltip.vue

```diff
@@ -65,6 +67,15 @@ onKeyStroke(
 )
 
 watch([isHovering, () => modifierKey.value, () => disabled.value], ([hovering, key, isDisabled]) => {
+  if (showOnTruncateOnly?.value) {
+    const targetElement = el?.value
+    const isElementTruncated = targetElement?.scrollWidth > targetElement?.clientWidth
```

Comment:
Nitpick: Instead of doing `el?.`, u can in L70 check if `el.value` is defined and remove all the `?.` in this if block